trappy/base.py

```diff
@@ -179,6 +179,12 @@ def append_data(self, time, comm, pid, cpu, line, data):
         self.line_array.append(line)
         self.data_array.append(data)
 
+    def conv_to_int(self, value):
```

Comment:
Sorry, can't resist bikeshedding here... shouldn't it be "maybe_parse_int" since we're only parsing an integer if it's an integer?